model-optimizer/mo/front_ng/pipeline.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+
+import logging as log
+from mo.front_ng.extractor import fe_user_data_repack
+from mo.middle.passes.infer import validate_batch_in_shape
+
+
+def moc_pipeline(argv: argparse.Namespace):
+    from ngraph import Dimension, PartialShape        # pylint: disable=no-name-in-module,import-error
+    from ngraph.utils.types import get_element_type   # pylint: disable=no-name-in-module,import-error
+    log.info('New MOC pipeline')
+    fem = argv.feManager
+    log.info(f'fem.availableFrontEnds: {str(fem.get_available_front_ends())}')
```

Comment:
The formatting strings with `f'...'` approach is not supported in Python3.5, so should be avoided. Please, refactor with regular `.format()` approach.

Reply:
Done, however 'pylint' suggests me now to change back to f""